src/components/molecules/nav/navigation.tsx

```diff
@@ -1,51 +1,48 @@
-/* eslint-disable */
-// This file was converted to .tsx without actually implementing typescript
-// @TODO: update this file to tsx and enable eslint
-import React, { Component } from 'react'
+import React, { Component, ReactNode } from 'react'
 import { Link } from 'gatsby'
 
 import styles from './navigation.module.css'
 
-import Menu from '../../../img/menu.inline.svg'
-import Close from '../../../img/close.inline.svg'
-import Down from '../../../img/down.inline.svg'
+import { Menu } from '../../../img/menu.inline'
```

Comment:
These don't really match the design exactly (Zeplin has chunky straight corners). Is that cool?

Reply:
Yeah, for now, I'm just going to use the existing svgs. We can update later, if we decide that's necessary